source/language/pulses.rst

```diff
@@ -29,10 +26,14 @@ instruction sequence on *physical* qubits, e.g.
 
    defcal rz(angle[20] theta) $0 { ... }
    defcal measure $0 -> bit { ... }
+   defcal measure_iq $q -> complex[32] { ... }
```

Comment:
`complex[float[32]]` to be compatible with the recently merged complex types PR.